ReleaseNotes.md

```diff
@@ -2,6 +2,10 @@
 
 These are the NuGet package releases. See also [npm Release Notes](ReleaseNotesNpm.md).
 
+## 2.10.0
+
+* Add `file-name-suffix` generator setting.
```

Comment:
There's been a number of commits since the package was published, so I'm not sure what all needs listed here.

Reply:
Hm, `Support generating Fastify plugins.` I guess. cc @mitchknife

Reply:
Oh good point. Yeah, that message works fine :+1:

Reply:
Also, sorry for not publishing a package earlier. That's bad form. 😁